src/System.Private.ServiceModel/src/Extensions/ReflectionExtensions.cs

```diff
@@ -92,7 +92,7 @@ public static bool IsValueType(this Type type)
         }
         public static InterfaceMapping GetInterfaceMap(this Type type, Type interfaceType)
         {
-            throw ExceptionHelper.PlatformNotSupported();
+            return type.GetTypeInfo().GetRuntimeInterfaceMap(interfaceType);
```

Comment:
To clarify: the test isn't passing for me right now, I need that one line change to get past the `PlatformNotSupportedException`.